### PR TITLE
Add unix support

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
Fixes #2 (and supersedes it)

Let me know if you have any requests for changes @seanyen

### CODE REVIEW

File: examples/conda-forge.yaml [hunk 1]
```diff
@@ -91,13 +97,18 @@ python3-netifaces:
 python-paramiko:
   conda-forge: [paramiko]
 sbcl:
```
Comment (seanyen) [inline examples/conda-forge.yaml:99]: can we do something like?

```
sbcl:
  conda-forge:
    win64: []
    unix: [sbcl]
```

File: examples/conda_build_config.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+# https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml
+
+pin_run_as_build:
+  boost:
+    max_pin: x.x.x
+  boost-cpp:
+    max_pin: x.x.x
+
+# blas
+libblas:
```
Comment (seanyen) [inline examples/conda_build_config.yaml:11]: wow, I like this way to lock down the version.

**Comment by seanyen** [comment]
@Tobias-Fischer It looks good to me in general. Thanks for iterating on that.

File: examples/conda-forge.yaml (hunk 1 above)
Comment (Tobias-Fischer) [reply examples/conda-forge.yaml:99]: We sure can. See https://github.com/ros-forge/vinca/pull/3/commits/a0d5b10c1937eee4fec2579c7c016c98e1223bd8

File: examples/conda_build_config.yaml (hunk 2 above)
Comment (Tobias-Fischer) [reply examples/conda_build_config.yaml:11]: Full credits to @wolfv!

**Comment by Tobias-Fischer** [comment]
Great to hear, thanks @seanyen. I addressed your comment.

**Comment by seanyen** [comment]
Thanks for the contribution!

**Comment by Tobias-Fischer** [comment]
Great, thanks for merging! What would the next steps look like to get CI running on this?

/cc @wolfv

**Pull request merged**